tiledb/api/src/array/schema.rs

```diff
@@ -1055,17 +953,17 @@ mod tests {
 
         // capacity change
         {
-            let cmp = start_schema(base.array_type())
-                .capacity((base.capacity() + 1) * 2)
+            let cmp = start_schema(base.array_type().unwrap())
+                .capacity((base.capacity().unwrap() + 1) * 2)
```

Comment:
Just a style fyi, I've found adding the `TileDBResult<()>` return value and just shoving an `Ok(())` as the last line of the test nicely allows for using `?` instead of having to type out `unwrap()` or `expect(...)`.

Reply:
I tried that for a bit, but what I like about this is that if the test fails it tells you the line number it failed on, whereas you don't get that with result.

Maybe if we put a stack trace in our error type :)

Reply:
Ah, that's a fair point.